source/initial-dns-seedlist-discovery.rst

```diff
@@ -0,0 +1,247 @@
+﻿.. role:: javascript(code)
+  :language: javascript
+
+==============================
+Initial DNS Seedlist discovery
+==============================
+
+:Spec-ticket: SPEC-878
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 0.3
```

Comment:
Shouldn't this start at 1.0 for a new spec?

Reply:
:+1: